pkg/reconciler/instances/istio/action.go

```diff
@@ -41,5 +44,36 @@ func (a *ReconcileAction) Run(version, profile string, config map[string]interfa
 		return errors.Wrap(err, "Could not patch MutatingWebhookConfiguration")
 	}
 
+	generated := generateNewManifestWithoutIstioOperatorFrom(manifest.Manifest)
+	fmt.Println(generated)
```

Comment:
Probably a left over? If not, please use the  provided `logger` instance from the `ActionContext`.